developer-docs-site/docs/reference/node_liveness_criteria.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Node Liveness Criteria"
+slug: " node_liveness_criteria"
+---
+
```

Comment:
Also, can you please change the filename to use dashes instead of underscores? Correct format, consistent with other filenames in the docs, is `node-liveness-criteria.md`. Thanks!